react/Gallery.js

```diff
@@ -61,6 +64,11 @@ class Gallery extends Component {
       [searchResult.galleryTwoColumns]: layoutMode === 'small' && mobile,
     })
 
+    const styleWithGap = {
+      ...style,
+      gridGap: gap,
```

Comment:
Check if `grid-gap` works on IE

Reply:
_Spoiler alert:_ it doesn't.

But so doesn't the grid anyway... By works I mean works